src/System.Buffers/tests/ArrayPool/ArrayPoolEventListener.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.Tracing;
+
+namespace System.Buffers.ArrayPool.Tests
+{
+    public sealed class ArrayPoolEventListener : EventListener
```

Comment:
See https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Diagnostics/Tracing/TestEventListener.cs and how we use it in tests for the other corefx EventSources we have.